plugin/phpactor.vim

```diff
@@ -37,8 +33,12 @@ if !exists('g:phpactorCompletionIgnoreCase')
     let g:phpactorCompletionIgnoreCase = 1
 endif
 
-if !exists('g:PhpactorQuickfixStrategy')
-    let g:PhpactorQuickfixStrategy = v:null
+if !exists('g:phpactorQuickfixStrategy')
+    let g:phpactorQuickfixStrategy = 'phpactor#quickfix#vim'
+endif
+
+if !exists('g:phpactorInputListStrategy')
+    let g:phpactorInputListStrategy = 'phpactor#input#list#inputlist'
```

Comment:
Moved these configurations to the main `phpactor.vim` so all configs are in the same place, and made them more explicitly callbacks.